README.md

```diff
@@ -89,18 +89,26 @@ optional arguments:
 ```
 * To install package dependencies using pip, run:
 ```
+    * If you logged in as root user *
     $ pip install requests
```

Comment:
Please correct the statement.

If you are logged in as `root` user, the following statements will not make any sense.
```
$ pip install requests
$ pip install click
```

If it is `root` then it should be:

```
# pip install requests
# pip install click
```

Reply:
@amolkahat made the changes as per suggestions.

Reply:
@shitalmule04 please read the comment carefully. I'm pointing you to the difference between `$` and `#`.